braintree/resource.py

```diff
@@ -17,7 +17,7 @@ def verify_keys(params, signature):
 
     @staticmethod
     def __flattened_params_keys(params, parent=None):
-        if isinstance(params, str):
+        if isinstance(params, str) or isinstance(params, unicode):
```

Comment:
this isn't Python 3 compatible as `unicode` doesn't exist in Python 3, you should do `isinstance(params, six.string_types)` (Using `six` makes this much easier, otherwise something akin to `string_types` will need defining)